test/index.html

```diff
@@ -19,37 +17,43 @@
 		<!-- <![endif]-->
 
 		<script>
-			window.tmpMoment = window.moment;
-			window.moment = undefined;
-
 			(function () {
-				var jQueryPath;
 
-				// Feature detect for jQuery version
+				/* - - - - - - - - - - - - - - - - - -
+					URL PARAMETER CONFIGURATION
+				- - - - - - - - - - - - - - - - - - */
+
+				// jQuery config
+				var jQueryPath;
+				// Feature detect
 				if('querySelector' in document
 					&& 'localStorage' in window
 					&& 'addEventListener' in window ) {
-					jQueryPath = 'bower_components/jquery/dist/jquery';
+					jQueryPath = 'bower_components/jquery/dist/jquery';	// v2.x
```

Comment:
Hrm... I'm slightly concerned it will be too easy/likely for this comment to be wrong. When we upgrade to jquery 3.x no one is going to remember to edit it. Then again, by then this whole file may be gone, so... idk.